### ISSUE TITLE
jQuery DataTables bug: "int parameter 'draw' is present but cannot be translated into a null value"

### ISSUE THREAD

**Issue by jrobble** [openmpf/openmpf#1237]
From the end user

> I'm continually getting this error in the ui:
>
>`DataTables warning: table id=jobTable - Ajax error. For more information about this error, please see http://datatables.net/tn/7` 

In the WFM log:

```
openmpf5_workflow-manager.1.jrc0iukayhln@some.host    | 2020-12-16 21:58:11,958 ERROR [http-nio-8080-exec-103] o.m.m.m.ControllerUncaughtExceptionHandler - Request for http://some.swarm.host/workflow-manager/jobs-paged raised an uncaught exception
openmpf5_workflow-manager.1.jrc0iukayhln@some.host    | java.lang.IllegalStateException: Optional int parameter 'draw' is present but cannot be translated into a null value due to being declared as a primitive type. Consider declaring it as object wrapper for the corresponding primitive type.
```

This error seems to be occurring when using the Job Status web UI because of a POST to http://openmpf-workflow-manager.swarm.va.dcap.fbi.gov/workflow-manager/jobs-paged where the “draw” parameter is present has no value. I’ve read that Spring MVC will attempt to set undefined parameters to “null”. In our REST endpoint code it seems that we attempt set a primitive int variable to that “null”, which is not valid. Some people online recommend using the Integer class instead of the primitive int.
 
Long story short, it seems that we can handle parameters a little better in our code, although, I’m not sure why the “draw” parameter would ever be set to null. We’re using the jQuery DataTables library: https://datatables.net/manual/server-side#Sent-parameters.
